interfaces/apparmor/backend.go

```diff
@@ -467,14 +487,22 @@ func (b *Backend) SetupMany(snaps []*snap.Info, confinement func(snapName string
 	}
 
 	if !fallback {
+		aaFlags := skipReadCache | conserveCPU
```

Comment:
Can we drop conserveCPU when in preseed mode?

Reply:
We probably could, but I'm not sure it's going to gain much / anything, the pre-bakinkg is done once to prepare an image, gaining a fraction of a second here is mostly irrelevant (contrary to saving it on actual firstboot/every boot).